java/code/src/com/suse/manager/webui/templates/groups/formulas.jade

```diff
@@ -1,9 +1,9 @@
 .spacewalk-toolbar-h1
   .spacewalk-toolbar
-    a(href="/rhn/groups/Delete.do?sgid=#{groupId}")
+    a(href="/rhn/groups/Delete.do?sgid=#{groupId}" class="btn btn-danger")
```

Comment:
Here and elsewhere in Jade files, it would be nice to use a consistent style for class names. For example we have `a(class="btn btn-default")`, but also `i.fa.fa-trash-o` in the same file. It would be preferable if we kept a consistent style and used one or the other. This is not an important detail though.

Reply:
If you see it in any specific file please let me know I can not find this use case.

Reply:
I tried to fix this in most places, but if we find any inconsistencies, let's fix them in a new PR.

Reply:
This issue is present in pretty much all touched Jade files in this PR. Allow me to rephrase so what I mean is clearer.  

In Jade files, you can create a div with `<div class="foo">` in two different ways: either as `.div` or as `div(class="foo")`. Existing code uses the first option throughout, so e.g. `i.fa.fa-trash-o`, but your additions use the other way, so e.g. `a(class="btn btn-danger")`. In this context there is no good reason to mix and match them, it would be preferable to use a consistent style throughout.

Reply:
Okay got it, my mistake. I will fix this.